lib/nostrum/struct/user.ex

```diff
@@ -63,6 +64,9 @@ defmodule Nostrum.Struct.User do
   @typedoc "The user's email"
   @type email :: String.t() | nil
 
+  @typedoc "The user's public flags"
+  @type public_flags :: Flags.t() | nil
```

Comment:
Although the `nil` here matches discord's documentation, it doesn't quite match with the PR's code, which suggests that this value will never be `nil` (if I'm not mistaken). I'd remove the `nil` here to simplify pattern matching.

Reply:
Addressed in aff5d4ca76ec14983946c67521b8f03850f0a78c